packages/oui-button/src/button.html

```diff
@@ -0,0 +1,20 @@
+<button
+  arial-label="{{$ctrl.text}}"
+  class="oui-button"
+  ng-attr-id="{{::$ctrl.id}}"
+  ng-attr-name="{{::$ctrl.name}}"
+  ng-attr-type="{{::$ctrl.type}}"
+  ng-class="{
+    'oui-button_primary': $ctrl.variant === 'primary',
+    'oui-button_secondary': $ctrl.variant === 'secondary',
+    'oui-button_link': $ctrl.variant === 'link',
+    'oui-button_icon-left': $ctrl.variantNav === 'previous',
+    'oui-button_icon-right': $ctrl.variantNav === 'next'
+  }"
+  ng-click="$ctrl.ngClick()"
```

Comment:
Using ng-click for a submit button should probably trigger a warning. We should force to use ng-submit on the form, it's a common problem. It could be a todo for the futur

Reply:
I don't know. The common problem is to use `ng-click` to submit a form. 
This thing must be verified with the PRs.

We can always have a case where we want to have a `ng-submit` and an action on the `ng-click` that is not a form submission.